### PR TITLE
fix profile route

### PR DESCRIPTION
**Pull request by esteban-gs**
## Dev Summary

Just cleaning up the profile route

## Test Plan

repro steps:

1. localhost:3000 
2. start ui and login with a valid user credential
expected: 

- when you click on the `My Profile` link, the route reads `/profile` instead of `/users/:userId`

## Resources

n/a

### CODE REVIEW

File: client/src/routes/routes.ts [hunk 1]
```diff
@@ -115,7 +115,7 @@ const routes: RouteMap = {
   User: {
     component: User,
     roles: ['OWNER', 'ADMIN'],
-    path: '/users/:userId',
+    path: '/profile',
```
Comment (jd2rogers2) [inline client/src/routes/routes.ts:118]: i think we'll want to show other users', not just the current user, profile page. so i think this isn't the right fix
we should instead fix the link to point to /users/[current_user.id]

**Review by jd2rogers2 — CHANGES_REQUESTED** [review]

File: client/src/routes/routes.ts (hunk 1 above)
Comment (esteban-gs) [reply client/src/routes/routes.ts:118]: So, the `user-id` path param determines the public user profile to fetch?

Comment (jd2rogers2) [reply client/src/routes/routes.ts:118]: that's how i imagined it/the most common pattern i've seen. if you have another you prefer then that's fine with me

Comment (jd2rogers2) [reply client/src/routes/routes.ts:118]: Sure, so i guess what you had originally is fine

So both urls point to the same react component
logic in there would be: 
Use url param user ID
If doesn't exist, use current user

**Review by jd2rogers2 — APPROVED** [review]

**Pull request merged**